Description: Upstream commit to version 3.7.1
             contact sheet: error if the theme file could 
             not be loaded. Fixes: #18
             06c39346fda502bd37429006d4822dd977995661
Author: Paolo Bacchilega (upstream)
Reviewed-by: Herbert Parentes Fortes Neto <hpfn@debian.org>
Last-Update: 2019-02-24
Index: gthumb/extensions/contact_sheet/dlg-contact-sheet.c
===================================================================
--- gthumb.orig/extensions/contact_sheet/dlg-contact-sheet.c
+++ gthumb/extensions/contact_sheet/dlg-contact-sheet.c
@@ -354,6 +354,7 @@ add_themes_from_dir (DialogData *data,
 			g_free (buffer);
 			g_object_unref (file);
 			g_object_unref (file_info);
+			continue;
 		}
 
 		theme = gth_contact_sheet_theme_new_from_key_file (key_file);
